modules/ROOT/pages/apikit-odatav4-expand-tutorial.adoc

```diff
@@ -0,0 +1,162 @@
+= Implement `$expand` leveraging flow reuse
+
+Since version 1.3.0 the module supports automatic routing for ease of
+implementing the `$expand` system query option on most common cases. Usage of
+this feature requires a little bit of preparation and configuration but it
+allows easy reuse of already implemented flows when processing `$expand`
+requests.
+
+For more information on the system query option see http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_SystemQueryOptionexpand[the OASIS specification].
+
+== A quick introduction to `$expand`
+
+The `$expand` operation allows API consumers to request inline representations
+for the related entities of query results. This lowers the amount of network
+roundtrips the consumer has to do when gathering relevant information.
+
+[NOTE]
+====
+
+Let's think of an example:
+
+* The `Employee` entity has a related `Manager` entity.
+* The `Employees` entity set is a collection of `Employee` entities.
+* The `Managers` entity set is a collection of `Manager` entities.
+* Our API is placed on `https://example.com/api/`
+
+Given this setup, the retrieval of employees with their managers requires:
+
+* `GET https://example.com/api/Employees` to retrieve a list of employees.
+* Gather the `Manager` ids through the `ReferentialConstraint` of their
```

Comment:
"ids" would be "IDs"? Do we explain before that each entity has an ID (though it is obvious)?

Reply:
Yes, unique identifier.